index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<title>Grow a Digitree</title>
+	<link rel="stylesheet" href="">
+</head>
+<body>
+
+	<form action="" method="get" accept-charset="utf-8">
```

Comment:
If there is no form action, then don't add the attribute. Also method="get" is the default, and accept-charset is unnecessary.

Simply use `<form>`